rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "nightly"
```

Comment:
I tried building pix-engine on Linux and I had this error:
error[E0554]: `#![feature]` may not be used on the stable release channel
   --> /home/vitaly/.cargo/registry/src/-678d78d9747a429f/thiserror-1.0.37/src/lib.rs:238:34
    |
238 | #![cfg_attr(provide_any, feature(provide_any))]
    |                                  ^^^^^^^^^^^

So I thought that you are using nightly. Now I think you use Windows, because on Windows I don't have this build failure. Anyway, I'm removing this change from the PR.

Reply:
i actually use macOS, but CI builds on all platforms. the error usually happens when your rust version is too far behind latest. the current main has a minimum supported rust version (MSRV) of 1.62.0 (i think the latest is 1.66.1)